backend/repository/config.go

```diff
@@ -56,50 +56,19 @@ type ThriftServiceMap map[string]map[string]*ThriftService
-	ThriftMethodName string                    `json:"thriftMethodName"`
-	WorkflowType     string                    `json:"workflowType"`
-	ClientID         string                    `json:"clientID"`
-	ClientMethod     string                    `json:"clientMethod"`
-	TestFixtures     []*TestStub               `json:"testFixtures"`
-	Middlewares      []*codegen.MiddlewareSpec `json:"middlewares"`
-	ReqHeaderMap     map[string]string         `json:"reqHeaderMap"`
-	ResHeaderMap     map[string]string         `json:"resHeaderMap"`
-}
-
-// TestStub saves stubbed requests/responses for an endpoint test.
-type TestStub struct {
-	TestName               string                 `json:"testName"`
-	EndpointID             string                 `json:"endpointId"`
-	HandlerID              string                 `json:"handlerId"`
-	EndpointRequest        map[string]interface{} `json:"endpointRequest"`
-	EndpointRequestString  string                 `json:"endpointRequestString"`
-	EndpointReqHeaders     map[string]string      `json:"endpointReqHeaders"`
-	EndpointReqHeaderKeys  []string               `json:"endpointReqHeaderKeys"`
-	EndpointResponse       map[string]interface{} `json:"endpointResponse"`
-	EndpointResponseString string                 `json:"endpointResponseString"`
-	EndpointResHeaders     map[string]string      `json:"endpointResHeaders"`
-	EndpointResHeaderKeys  []string               `json:"endpointResHeaderKeys"`
-	ClientStubs            []ClientStub           `json:"clientStubs"`
-	TestServiceName        string                 `json:"testServiceName"`
-}
-
-// ClientStub saves stubbed client request/response for an endpoint test.
-type ClientStub struct {
-	ClientID             string                 `json:"clientId"`
-	ClientMethod         string                 `json:"clientMethod"`
-	ClientRequest        map[string]interface{} `json:"clientRequest"`
-	ClientRequestString  string                 `json:"clientRequestString"`
-	ClientReqHeaders     map[string]string      `json:"clientReqHeaders"`
-	ClientReqHeaderKeys  []string               `json:"clientReqHeaderKeys"`
-	ClientResponse       map[string]interface{} `json:"clientResponse"`
-	ClientResponseString string                 `json:"clientResponseString"`
-	ClientResHeaders     map[string]string      `json:"clientResHeaders"`
-	ClientResHeaderKeys  []string               `json:"clientResHeaderKeys"`
+	ID               string                       `json:"endpointId"`
+	Type             ProtocolType                 `json:"endpointType"`
+	HandleID         string                       `json:"handleId"`
+	ThriftFile       string                       `json:"thriftFile"`
+	ThriftFileSha    string                       `json:"thriftFileSha,omitempty"`
+	ThriftMethodName string                       `json:"thriftMethodName"`
+	WorkflowType     string                       `json:"workflowType"`
+	ClientID         string                       `json:"clientId"`
```

Comment:
Renaming the json tag(e.g. ClientID) is not essential to this PR and is a breaking change. Please make a separate PR to rename the tags.

Reply:
I thought we agreed to introduce breaking change for the consistency of `go field ID and json tag Id`?
This PR already contains breaking changes would it make sense to reduce the # of breaking changes?
I feel strong about consistent naming

Reply:
sounds good.